docs/babel.rst

```diff
@@ -0,0 +1,64 @@
+
+HistFitter
+----------
+
+In order to go from `HistFitter` to `pyhf`, the first step is to extract out the `HistFactory` workspaces. Assuming you have an existing configuration file, ``config.py``, you likely run an exclusion fit like so:
+
+.. code:: bash
+
+  HistFitter.py -f -D "before,after,corrMatrix" -F excl config.py
+
+The name of output workspace files depends on four parameters you define in your ``config.py``:
+
+- ``analysisName`` is from ``configMgr.analysisName``,
+- ``prefix`` is defined in ``configMgr.addFitConfig({prefix})``, and
+- ``measurementName`` is the first measurement you define via ``fitConfig.addMeasurement(name={measurementName},...)``
+- ``channelName`` are the names of channels you define via ``fitConfig.addChannel("cuts", [{channelName}], ...)``
+- ``cachePath`` is where `HistFitter` stores the cached histograms, defined by ``configMgr.histCacheFile`` which defaults to ``data/{analysisName}.root``
+
+To dump the HistFactory workspace, you will modify the above to skip the fit ``-f`` and plotting ``-D`` so you end up with
+
+.. code:: bash
+
+  HistFitter.py -wx -F excl config.py
+
+The ``-x`` flag tells `HistFitter` to dump the XML files into ``config/{analysisName}/``, with the top-level file being ``{prefix}.xml`` and all other files being ``{prefix}_{channelName}_cuts.xml``. The ``-w`` flag tells `HistFitter` to (re)create the `HistFactory` workspace stored in ``results/{analysisName}/{prefix}_combined_{measurementName}.root``.
+
+Typically, ``prefix = 'FitConfig'`` and ``measurementName = 'NormalMeasurement'``. For example, if the following exists in your ``config.py``
+
+.. code:: python
+
+  from configManager import configMgr
+  # ...
+  configMgr.analysisName = '3b_tag21.2.27-1_RW_ExpSyst_36100_multibin_bkg'
+  configMgr.histCacheFile = 'cache/{0:s}.root'.format(configMgr.analysisName)
+  # ...
+  fitConfig = configMgr.addFitConfig("Excl")
+  # ...
+  channel = fitConfig.addChannel("cuts", ['SR_0L'], 1, 0.5, 1.5)
+  # ...
+  meas1=fitConfig.addMeasurement(name="DefaultMeasurement",lumi=1.0,lumiErr=0.029)
+  meas1.addPOI("mu_SIG1")
+  # ...
+  meas2=fitConfig.addMeasurement(name="DefaultMeasurement",lumi=1.0,lumiErr=0.029)
+  meas2.addPOI("mu_SIG2")
+
+Then, you expect the following files to be made:
+
+- ``config/3b_tag21.2.27-1_RW_ExpSyst_36100_multibin_bkg/Excl.xml``
+- ``config/3b_tag21.2.27-1_RW_ExpSyst_36100_multibin_bkg/Excl_SR_0L_cuts.xml``
+- ``cache/3b_tag21.2.27-1_RW_ExpSyst_36100_multibin_bkg.root``
+- ``results/3b_tag21.2.27-1_RW_ExpSyst_36100_multibin_bkg/Excl_combined_DefaultMeasurement.root``
+
+These are all the files you need in order to use `pyhf xml2json <cli.html#pyhf-xml2json>`_. At this point, you could run
+
+.. code:: bash
+
+    pyhf xml2json config/3b_tag21.2.27-1_RW_ExpSyst_36100_multibin_bkg/Excl.xml
+
+which will read all of the XML files and load the histogram data from the histogram cache.
```

Comment:
While this will be clear soon after running, and is already explicitly stated from the Command Line API section in the docs, maybe it would be mentioning that as `xml2json` doesn't have a default value for `--output-file` they should provide this option here.

Reply:
Not in scope. This guide is only to get to the point where they can run `pyhf xml2json`.

Reply:
Why is that out of scope? Maybe I'm missing what you want the user of these docs to do, but if I was running through this for the first time I would expect to end with a file that I can do things with, not output to screen. You already are giving them an example run command.

Reply:
If the user does not know how to redirect output to a file, I don't think they should be following these instructions blindly...

Reply:
Additionally, these docs won't be tested, so having only the necessary information (and ignoring the CLI options that could change in the future) is going to make these robust/future-proof.